lms/djangoapps/email_marketing/tasks.py

```diff
@@ -294,6 +300,99 @@ def _get_course_content(course_url, sailthru_client, email_config):
     return response
 
 
+def _get_or_create_user_list_for_site(sailthru_client, site=None, default_list_name=None):
+    """
+    Get the user list name from cache if exists else create one and return the name,
+    callers of this function should perform the enabled check of email config.
+    :param: sailthru_client
+    :param: site
+    :param: default_list_name
+    :return: list name if exists or created else return None
+    """
+    if site and site.get('id') != settings.SITE_ID:
+        list_name = site.get('domain').replace(".", "_") + "_user_list"
```

Comment:
This statement is dangerous. If `site.get('domain')` returns `None`, this is an exception. Maybe provide a default value?